src/app/routes/onDemandRadio/getInitialData/index.js

```diff
@@ -60,6 +70,9 @@ export default async ({ path: pathname }) => {
         releaseDateTimeStamp: getReleaseDateTimeStamp(json),
         pageTitle: getPageTitle(json),
         pageIdentifier: getPageIdentifier(json),
+        promoBrandTitle: getPromoBrandTitle(json),
+        durationISO8601: getDurationISO8601(json),
+        imageUrl: getPlaceholderImageUrl(getImageUrl(json)),
```

Comment:
Can we just have          `imageUrl: getImageUrl(json),` and move the placeholder image logic to that function instead?